frontend/simple/js/state.js

```diff
@@ -89,7 +89,10 @@ const mutations = {
 // https://vuex.vuejs.org/en/modules.html
 const getters = {
   currentGroup (state) {
```

Comment:
To avoid confusion I recommend refactoring project to rename this to `currentGroupState` since that's what it's returning.